.golangci.yml

```diff
@@ -18,6 +18,11 @@ run:
     - ^proto
     - ^.git
 linters-settings:
+  exhaustive:
+    # Presence of "default" case in switch statements satisfies exhaustiveness,
+    # even if all enum members are not listed.
+    # Default: false
+    default-signifies-exhaustive: true
```

Comment:
@tdeebswihart FYI, I added this config. the errors on `switch`s with `default` case did not make much sense to me.

Reply:
Actually reverting the config in here: https://github.com/temporalio/temporal/pull/5722